autoload/neomake/quickfix.vim

```diff
@@ -33,10 +33,18 @@ function! s:cursor_moved() abort
         endif
 
         silent! call matchdelete(s:cursor_match_id)
-        call matchaddpos('neomakeCursorListNr',
-                    \ [[line('.'), (b:neomake_start_col - b:neomake_number_len) + 2, b:neomake_number_len]],
-                    \ s:cursor_match_id,
-                    \ s:cursor_match_id)
+        if exists('*matchaddpos')
+            call matchaddpos('neomakeCursorListNr',
+                        \ [[line('.'), (b:neomake_start_col - b:neomake_number_len) + 2, b:neomake_number_len]],
+                        \ s:cursor_match_id,
+                        \ s:cursor_match_id)
+        else
+            call matchadd('neomakeCursorListNr',
+                        \  '\%' . line('.') . 'c'
+                        \. '\%' . ((b:neomake_start_col - b:neomake_number_len) + 2) . 'c'
+                        \. '.\{' . b:neomake_number_len . '}',
+                        \ s:cursor_match_id, s:cursor_match_id)
+        endif
```

Comment:
It's used `autoload/neomake/highlights.vim` also, so there could be a wrapper.
OTOH I've noticed that using `matchadd` (with `\%Xv` (not `c` like above)) seems to be the sanest currently, given that other methods use byte offsets.  See https://github.com/neovim/neovim/issues/6270#issuecomment-286403626.
So a wrapper would make it easier to handle this in the future, but it's fine to keep this here for now.
However, I think `v` with `matchadd` might be the best regarding multibyte strings.
If keeping `matchaddpos`, then `nvim_buf_add_highlight` might be an option for Neovim, too.